lib/shopify_cli/theme/dev_server/hot_reload/liquid_css_reloader.rb

```diff
@@ -0,0 +1,61 @@
+# frozen_string_literal: true
+
+module ShopifyCLI
+  module Theme
+    module DevServer
+      class HotReload
+        class LiquidCssReloader
```

Comment:
Nitpick, but nothing in this class is specific to `.css.liquid` file right? Could be renamed to something more generic?

```suggestion
        class RemoteFileReloader
```

Reply:
Excellent suggestion 🚀 Thanks, @macournoyer!